sdk/trace/batch_span_processor.go

```diff
@@ -145,6 +145,10 @@ func (bsp *batchSpanProcessor) OnEnd(s ReadOnlySpan) {
 func (bsp *batchSpanProcessor) Shutdown(ctx context.Context) error {
 	var err error
 	bsp.stopOnce.Do(func() {
+		err = bsp.ForceFlush(ctx)
+		if err != nil {
+			return
```

Comment:
```suggestion
			return err
```

Reply:
That `return` returns from the anonymous `Do` parameter, not from `Shutdown`.

Reply:
You're right, my mistake.